packages/redux/README.md

```diff
@@ -112,3 +153,14 @@ class App extends Component {
 
 export default App;
 ```
+
+### Example
+You can also test a demo application by uploading it to [this link](https://download-directory.github.io/?url=https://github.com/AxaGuilDEv/react-oidc/tree/master/examples/redux) or cloning [the repository](https://github.com/AxaGuilDEv/react-oidc.git) (examples / redux directory).
+Then you just need to run a
+``` shell
+yarn install
```

Comment:
We use npm everywhere inside the documentation

Reply:
changed, thanks for review